.circleci/config.yml

```diff
@@ -114,11 +115,11 @@ commands:
       - run:
           name: "Install CUDA"
           command: |
-            wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1604/x86_64/cuda-ubuntu1604.pin
-            sudo mv cuda-ubuntu1604.pin /etc/apt/preferences.d/cuda-repository-pin-600
-            wget http://developer.download.nvidia.com/compute/cuda/10.2/Prod/local_installers/cuda-repo-ubuntu1604-10-2-local-10.2.89-440.33.01_1.0-1_amd64.deb
-            sudo dpkg -i cuda-repo-ubuntu1604-10-2-local-10.2.89-440.33.01_1.0-1_amd64.deb
-            sudo apt-key add /var/cuda-repo-10-2-local-10.2.89-440.33.01/7fa2af80.pub
+            wget https://developer.download.nvidia.com/compute/cuda/repos/ubuntu2004/x86_64/cuda-ubuntu2004.pin
+            sudo mv cuda-ubuntu2004.pin /etc/apt/preferences.d/cuda-repository-pin-600
```

Comment:
general question: Did you get a notification that we need this update ? I wonder if there was an error or notification that asks us to make the change.

Reply:
Yes, I got this email from CircleCI:
<img width="790" alt="Screen Shot 2022-03-09 at 3 03 51 PM" src="https://user-images.githubusercontent.com/11067177/157528204-8e8d44da-5711-4441-9e73-f62ac94f3fce.png">